resources/js/admin/settings/components/SettingsListing.vue

```diff
@@ -63,34 +63,36 @@
                   <i class="fa-lg fas fa-edit"></i>
                 </b-button>
             </span>
-            <b-button 
-              :aria-label="$t('Copy to Clipboard')"
-              v-uni-aria-describedby="row.item.id.toString()"
-              @click="onCopy(row)"
-              variant="link"
-              v-b-tooltip.hover
-              :title="$t('Copy to Clipboard')">
-                <i class="fa-lg fas fa-copy"></i>
-              </b-button>
-
-            <span v-b-tooltip.hover v-if="!['boolean', 'object', 'button'].includes(row.item.format)" :title="$t('Clear')">
-              <b-button 
-              :aria-label="$t('Clear')"
-              v-uni-aria-describedby="row.item.id.toString()"
-              :disabled="disableClear(row.item)" 
-              @click="onClear(row)" 
-              variant="link" 
-             >
-                <i class="fa-lg fas fa-trash-alt"></i>
-              </b-button>
-            </span>
-            <span v-else class="invisible">
+            <span v-if="row.item.key !== 'sso.default.login'" >
               <b-button 
+                :aria-label="$t('Copy to Clipboard')"
+                v-uni-aria-describedby="row.item.id.toString()"
+                @click="onCopy(row)"
+                variant="link"
+                v-b-tooltip.hover
+                :title="$t('Copy to Clipboard')">
+                  <i class="fa-lg fas fa-copy"></i>
```

Comment:
If the tag doesn't have any value must be closed in the same.
in this case:
<i class="fa-lg fas fa-copy" />